### ISSUE TITLE
request body in DELETE ?

### ISSUE THREAD

**Issue by sarangsbabu367** [p1c2u/openapi-core#244]
Does openapi support `request-body` for DELETE request ?. \
I am getting `None` after validation.

**Comment by p1c2u** [comment]
@sarangsbabu367 
It vary integration you use. The lib itself doesn't retrieve request body just process it. Do you get request body before validation?

**Comment by sarangsbabu367** [comment]
@p1c2u Yes i am getting. But i think there is some issue with path eventhough my path is of form `/users/{id}/relationships/accounts`, when i tried to log the path. I got path as `/users/{id}` and in template i got remaining part separated.

**Comment by p1c2u** [comment]
It can be related to [this](https://github.com/p1c2u/openapi-core/issues/226) issue then
